examples/create-Redirection/api_create_redirection.md

```diff
@@ -0,0 +1,125 @@
+How to get web hosting capabilities using php wrapper?
+----------------------------------------------------
+
+This documentation will help you to create an HTTP redirection from a subdomain to another domain. Following script include DNS record check and delete record if their will conflict with your redirection!
+
+## Requirements
+
+- Having PHP 5.2+
+- Having a dns record at OVH
+
+## Download PHP wrapper
+
+- Download the latest release **with dependencies** on github: https://github.com/ovh/php-ovh/releases
+
+```bash
+# When this article was written, latest version is 2.0.0
+wget https://github.com/ovh/php-ovh/releases/download/v2.0.0/php-ovh-2.0.0-with-dependencies.tar.gz
+```
+
+- Extract it into a folder
+
+```bash
+tar xzvf php-ovh-2.0.0-with-dependencies.tar.gz 
+```
+
+- Create a new token
+You can create a new token using these url: [https://api.ovh.com/createToken/?GET=/domain/zone/*&POST=/domain/zone/*&DELETE=/domain/zone/*](https://api.ovh.com/createToken/?GET=/domain/zone/*&POST=/domain/zone/*&DELETE=/domain/zone/*). Keep application key, application secret and consumer key to complete the script.
+
+Be warn, this token is only validated for this script and to use **/domain/zone/\*** APIs.
+If you need a more generic token, you had to change right field.
```

Comment:
My problem here is that's customer could customize data like DNS zone and IDs. I prefere to be generic.